src/main/webapp/WEB-INF/views/customerSearchScreen.html

```diff
@@ -0,0 +1,146 @@
+
+
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="utf-8">
+    <title>Customer Search Screen</title>
```

Comment:
Same here, I think we could make it Customer Service Search, or something along those lines?